compiler/rustc_interface/src/util.rs

```diff
@@ -219,13 +224,13 @@ fn load_backend_from_dylib(path: &Path) -> fn() -> Box<dyn CodegenBackend> {
     }
 }
 
-pub fn get_codegen_backend(sess: &Session) -> Box<dyn CodegenBackend> {
+pub fn get_codegen_backend(codegen_name: Option<&str>) -> Box<dyn CodegenBackend> {
```

Comment:
I would take `&config::Options` here, so that the `.debugging_opts.codegen_backend` part stays in this function, but I guess it doesn't matter much.